app/questionnaire/router.py

```diff
@@ -162,6 +166,9 @@ def is_survey_complete(self):
 
         return True
 
+    def get_location_for_hub_when_section_complete(self, routing_path) -> Location:
```

Comment:
I don't think it's clear what this method does as it sounds like it would return the url of the hub. How about `get_section_return_location_when_section_complete`?